parser.y

```diff
@@ -51,10 +51,14 @@ extern std::unique_ptr<AstNode> program;
 %token IF ELSE
 %token EQ NE LE GE
 %token DO WHILE FOR
+// increment (PP: ++) and decrement (MM: --)
+%token PP MM
```

Comment:
```suggestion
%token INC DEC
```
How about using `INC` (increment) and `DEC` (decrement) instead of `PP` and `MM`? The latter might be a bit cryptic. :laughing:
(Or, to be consistent with the name of the node, use `INCR` and `DECR`.)